docs/collections/documents/tutorials/query-doc-store-api.md

```diff
@@ -1,207 +1,258 @@
 ---
 sidebar_position: 70
-title: Query Docs as API
+title: REST API Query Doc Store Example
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Prerequisites from '../../../_partials/_prerequisites-api-key.md';
+import Steps from '../../../_partials/_api-example-steps.md';
 
-Modern applications need to be highly responsive, always online, and able to access data instantly across the globe. At the same time, they need to be deployed on datacenters close to their users. Macrometa global data network (GDN) is a real-time materialized view engine that provides instant data to applications and APIs in a simple interface.
+This page shows you how to query documents stored in a document store collection using the Macrometa API. For more information about using Macrometa APIs, refer to [APIs](../../../api-docs/index.md).
```

Comment:
@diana-macrometa I have updated the code samples to working ones with same output format.. But the thing is this whole section is about "QueryWorkers" using API, basically CRUD operations with QueryWorkers using API (python and javascript examples).. so we need to update the description/content/title of this page

Reply:
I updated the intro. Does the title also need changing?

Also, now that I understand this content better, I'm considering moving it to the Query Workers section. It seems like the "query worker" part of this example is more important than the fact that it is in a document collection. What do you think?

Reply:
@diana-macrometa this whole example is doing CRUD operations on QueryWorkers and how you can use query workers using RestApi.. and not how you can use c8ql queries (yes in the end queryworkers are just queries but we want to focus on query worker part and yes we can move it to query workers section and rename everything)... I still don't feel the description is right.. But maybe you can have a fresh look from a Query Workers point of view

Reply:
Took a stab at rewriting and updated the comments in the code to read "query workers" instead of "queries."

Reply:
@diana-macrometa Looks good ..but what do you think abut the Heading/title of the page and the file name (.md) and which section it should go in?

Reply:
Do you have any suggestions?

Reply:
@diana-macrometa I checked this out. I think moving it and having the title be something like `REST API Query Worker create, read, update and delete` makes sense

Reply:
Updated title and moved to Query Workers section.